### ISSUE TITLE
Delphi 10.4 and up

### ISSUE THREAD

**Issue by AVDeveloppement** [micha137/dspack-continued-mirror-for-delphinus#4]
Hello, 
It compil on Delphi 10.4 and 11 but can't install design package.
Get error modul not found
How debug package install for fix that ? I don't find :(

**Comment by micha137** [comment]
Hi,
thanks for the report, but I the error description is not helpful.  Can you post the complete error message?

**Comment by AVDeveloppement** [comment]
`---------------------------
Erreur
---------------------------
Impossible de charger le package C:\Users\Public\Documents\Embarcadero\Studio\22.0\Bpl\DSPack_D280.bpl.
Le module spécifié est introuvable.
---------------------------
OK   
---------------------------
`
Copy Past, first time i see that when install a package, i use community version.
And of course package are in dir. 
Thanks for the answer

**Comment by micha137** [comment]
This repository doesn't include a package for Delphi 11 yet.  You created it by yourself using LIBSUFFIX $(Auto), which is good.  Maybe you only created the new package after installing the 10.4 package into 11?  There might be old files DSPack_*.{bpl,dcp} around, leading to confusion.  Delete them and then make sure to compile the runtime package DSPack_R*.{bpl,dcp} for 32 bit before trying to install the design time package.

**Comment by AVDeveloppement** [comment]
Exactly, i tried that. I make new win 11 vm fresh install of Alexandria and i gey same error. On Rio it's ok

**Comment by micha137** [comment]
Googling for your error message lets me believe the problem is neither D11 nor DSPack specific: https://stackoverflow.com/questions/39726735/cant-load-package-s-error-while-installing-a-package
I think I can't help you any further.  But let me know what was the issue..
